docs/src/appendix/forge/clean-cache.md

```diff
@@ -0,0 +1,7 @@
+# `snforge clean-cache`
+
+Clean Forge cache directory.
+
+## `-h`, `--help`
+
+Print help.
```

Comment:
Does it make sense to have help flag for an empty command?

Reply:
It is provided by clap, not my choice